components/agenda/src/Agenda.svelte

```diff
@@ -81,7 +81,7 @@
     color_by: "event",
     condensed_view: false,
     eagerly_fetch_events: true,
-    end_minute: 1440,
+    end_minute: 1000,
```

Comment:
Why did this change? 1440 is 24 hours

Reply:
The problem with 24 hours as the end minute is the 15 min meetings are no longer visible.
![image](https://user-images.githubusercontent.com/34139730/145625302-0232d4fa-375a-4b28-8c7a-e661a62bfd37.png)

In order to fix this, we reduce the amount of hours initially visible so that the 15 min meeting is always visible and set that as the [max](https://github.com/nylas/components/pull/263/files#diff-d89f23d5aacd3516639ccaf42ec8d6f4735f4c6498f775bf36646b997a13ff14R597) zoom out view.
![image](https://user-images.githubusercontent.com/34139730/145625422-eca2784d-e219-4a4a-8557-28b1293913d4.png)

Reply:
Gotcha, makes sense

Reply:
Struggling with this change and would prefer we not do it:
- a max of 1000 minutes shown means that, even on an enormous vertical monitor or something, we'd only be showing to ~5pm. If we're going to do this, we should make it based on clientHeight (see below)
- We already have the concept of `minEventHeight` in Agenda; that should handle this exact case and cause overflow / force Condensed mode. I am not sure if it's working as intended currently if this bug was found, but it stipulates a minimum event height of 30px and overflows in that case.

Reply:
@philrenaud 
I agree with 1000 not being hardcoded, it should be relative to the size of the main component.

`minEventHeight` seem to work because the event content won't shrink passed the `minEventHeight` but the overflow still doesn't allow for the event to be visible because it's white font on white bg.
![image](https://user-images.githubusercontent.com/34139730/145633873-e6d530d3-1024-4dc1-8d9b-d5c1594a5c5f.png)